cookbooks/env.rst

```diff
@@ -173,11 +173,11 @@ will evaluate to an empty string like any other unset variable:
 * ``SYMFONY_ENVIRONMENT``: The name of the environment generated by the name of
   the Git branch.
 
-* ``SYMFONY_DEFAULT_ROUTE_URL``: The default endpoint serving your application.
-  It can be used to avoid hard-coding domains that can be used to reach
-  non-production environments. Parts of the URL (scheme, domain, port and path)
-  are also exposed each in their own variable whose name starts with
-  ``SYMFONY_DEFAULT_ROUTE_`` followed by the name of the part.
+* ``SYMFONY_PROJECT_DEFAULT_ROUTE_URL``: The default endpoint serving your
+  project. It can be used to avoid hard-coding domains that can be used to
+  reach non-production environments. Parts of the URL (scheme, domain, port and
+  path) are also exposed each in their own variable whose name starts with
+  ``SYMFONY_PROJECT_DEFAULT_ROUTE_`` followed by the name of the part.
```

Comment:
Do we list the part names somewhere?

Reply:
implicitly:
> scheme, domain, port and path

Reply:
ok, maybe we can change it to:

Parts of the URL are also exposed as their own variables whose name starts with ``SYMFONY_PROJECT_DEFAULT_ROUTE_`` followed by the name of the part (``SCHEME``, ``DOMAIN``, ``PORT``, and ``PATH``).